### PR TITLE
added referred url from event data

### PR DESCRIPTION
**Pull request by gvidoou**

### CODE REVIEW

**Review by Bukashk0zzz — CHANGES_REQUESTED** [review]
That's wrong place. you need to add this in https://github.com/stape-io/facebook-tag/blob/main/template.js#L599

and add if for check if it exist.

File: template.tpl [hunk 1]
```diff
@@ -1377,6 +1377,8 @@ function addAppData(eventData, mappedData) {
   if (eventData.windows_attribution_id)
     mappedData.app_data.windows_attribution_id =
       eventData.windows_attribution_id;
+  if (eventData.referrer_url)
```
Comment (soniar4i) [inline template.tpl:1380]: Shouldn't this check be against page_referrer?

File: template.js [hunk 2]
```diff
@@ -643,6 +643,8 @@ function addAppData(eventData, mappedData) {
   if (eventData.windows_attribution_id)
     mappedData.app_data.windows_attribution_id =
       eventData.windows_attribution_id;
+  if (eventData.referrer_url)
```
Comment (soniar4i) [inline template.js:646]: Shouldn't this check be against page_referrer?

Comment (Bukashk0zzz) [reply template.js:646]: it should

**Pull request merged**